enos/enos-samples-ce-build.hcl

```diff
@@ -200,6 +217,7 @@ sample "build_ce_linux_amd64_zip" {
       arch            = ["amd64"]
       artifact_type   = ["bundle"]
       artifact_source = ["crt"]
+      consul_edition  = ["ce"]
```

Comment:
For zip bundles I think we ought to only test on `ubuntu` and `amz2` while leaving all of the distros for the system packages. The idea is that when we're testing our zip bundles we'll favor testing on distros that don't have vendor markup.